packages/@okta/vuepress-site/docs/reference/api/oidc/index.md

```diff
@@ -299,7 +299,7 @@ Based on the scopes requested. Generally speaking, the scopes specified in a req
 | :----------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | invalid_client           | The specified `client_id` wasn't found.                                                                                                                                                                      |
```

Comment:
General comment: Should the sentences in the Details column be in the present tense?
wasn't found > isn't found
was invalid > is invalid